src/engraving/tests/parts_tests.cpp

```diff
@@ -1175,3 +1177,42 @@ TEST_F(Engraving_PartsTests, staffStyles)
 }
 
 #endif
+
+TEST_F(Engraving_PartsTests, staffIdsFromRange)
```

Comment:
I think PartsTests is not the correct place for this. PartsTests is actually about generating _`Excerpts`_. Maybe create a new test suite about `Parts` or about staves? Or about methods of the `Score` class?

Reply:
Added a separate file (scoreutils_tests)